### DISCUSSION TITLE
Open to a fork and leadership change?

### DISCUSSION THREAD

**Discussion by cdbattags** [nginxinc/ngx-rust#64] [General]
Anyone else watching this repo potentially want to go for a proper 1.0 on this?

I love Nginx but now with F5/Kong ownership it looks like this concept is a moonshot.

I'm currently the maintainer of https://luarocks.org/modules/cdbattags/lua-resty-jwt and have experience in the OpenResty.

I believe options other than Lua are a necessity to stay competitive in the open-source arena.

Thoughts?

**Comment by f5yacobucci** [comment]
We've rebooted this effort, changed direction and maintenance efforts. We're seeking community feedback and look forward to working together to bring these ideas forward.
